Source/WebKit/UIProcess/mac/WebContextMenuProxyMac.mm

```diff
@@ -446,8 +446,8 @@ static void getStandardShareMenuItem(NSArray *items, void (^completionHandler)(N
         return;
     }
 
-    auto sharingServicePicker = adoptNS([[NSSharingServicePicker alloc] initWithItems:items.get()]);
-    if ([sharingServicePicker respondsToSelector:@selector(standardShareMenuItem)]) {
+    if (@available(macOS 13.0, *)) {
```

Comment:
We don't use `@available` in WebKit. If you really want to remove this `respondsToSelector` check, you could add a new flag to `wtf/PlatformHave.h`.

Reply:
It appears elsewhere in the code, but yeah I'll remove it. I strongly feel like this is the best way to approach it though

Reply:
There are instances in third-party libraries, but not in WebCore/WebKit itself.

Reply:
Done! @pxlcoder